test/lib/heroku-api-test.js

```diff
@@ -57,9 +57,10 @@ describe('heroku-api', function () {
       const pipeline = '123-abc'
       const number = 1
       const testRun = { number }
-      const api = nock(`https://api.heroku.com`)
+      const api = nock(`https://api.heroku.com`, {reqheaders: {Accept: 'application/vnd.heroku+json; version=3.ci'}})
         .get(`/pipelines/${pipeline}/test-runs/${number}`)
-        .matchHeader('Accept', 'application/vnd.heroku+json; version=3.ci')
+        // nock node 8 bug https://github.com/node-nock/nock/issues/925
+        // .matchHeader('Accept', 'application/vnd.heroku+json; version=3.ci')
```

Comment:
so a refactor I would suggest but won't work with the new syntax is to use nock in a beforeEach filter like we do here: https://github.com/heroku/heroku-cli-status/blob/master/src/commands/status.test.js#L9

that saves a couple of lines of code in each test and ensures api.done is always called